CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+### v2.0.0-beta3 -> v2.0.0-beta4
+
+#### Features
```

Comment:
@mboersma thank you, yes missed the tidying up in this one in my hurry... should be cleaned up now!